src/systemtest/resources/system-test.properties

```diff
@@ -0,0 +1,21 @@
+application.name=eiffel-intelligence
+
+# Test Framework logging level
+logging.level: INFO
+
+######## EI Front-End
+ei.frontend.url:http://localhost:8077
+
+######## EI Back-End
+ei.backend.url.artifact:http://localhost:8070
```

Comment:
Shouldn't these parameter be using the internal hostnames and inernal ports:
ei-backend-artifact:8080, ei-backend-sourcechanget:8080,,, and so on.

Reply:
or $EI_BACKEND_ARTIFACT_HOSTNAME:$EI_BACKEND_ARTIFACT_INTERNAL_PORT